bin/publish.js

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env node
```

Comment:
Should we link the glimmer file in a comment as the source for this?

Reply:
I did take some inspiration, so why not :)